### ISSUE TITLE
branch cpg-guidance is using unpublished package

### ISSUE THREAD

**Issue by bdc-ehealth** [hl7-be/pss#69]
@JDW2021 @NISN-SMALS ,

branch cpg-guidance, based on the examples provided by @costateixeira is referring to an unpublished package:

![image](https://github.com/user-attachments/assets/11720bb0-37aa-4e02-9071-4d6817ce3858)

This cannot be used as the input for an official publication.

**Comment by costateixeira** [comment]
Yes, this is known and mentioned in the issue. It's not a new issue.

For information: The timelines for publishing CPG are FHIR management Group
this week and TSC Monday, and hopefully publication shortly thereafter.
If that is not adequate with the timelines we have, we will need to create
the same extensions (that will have little impact on the implementers with
the canonicals being belgian, not HL7 International, which I would prefer).
In both cases we keep alignment to the standard.

**Comment by costateixeira** [comment]
CPG is now published:
https://hl7.org/fhir/uv/cpg

**Comment by bdc-ehealth** [comment]
We especially need the reference to https://hl7.org/fhir/uv/cpg/OperationDefinition-cpg-plandefinition-apply.html here, because it deviates from https://hl7.org/fhir/R4/plandefinition-operation-apply.html

**Comment by bdc-ehealth** [comment]
I will merge this to earlyadopter, and from there it can be approved by the HL7 PSS workgroup (move to master) and be a releasecandidate (move to releasecandidate)

**Comment by costateixeira** [comment]
I prefer the WG to approve the content as is in the CP guidance. If you
merge to earlyadopter, i think that should be cleared first
